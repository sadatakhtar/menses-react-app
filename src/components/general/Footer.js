import React from 'react'
import '../../styles/Footer.css'

const Footer = () => {
  return (
    <div>
      <footer>
        <p>&copy; 2023 CycleSync. All rights reserved.</p>
      </footer>
    </div>
  )
}

export default Footer