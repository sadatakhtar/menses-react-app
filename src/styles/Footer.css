footer {
    background-color: black;
    height: 10vh;
    color: antiquewhite;
    display: flex;
    justify-content: center;
    align-items: center;
    font-size:small;
    position: fixed;
    bottom: 0;
    width: 100%;
    
}