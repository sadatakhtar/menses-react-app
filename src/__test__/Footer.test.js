import React from "react";
import { render, screen } from "@testing-library/react";
import Footer from "../components/general/Footer";

describe("Footer Component", () => {
  it("Footer snapshot created without errors", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });

  it("displays the copyright text", () => {
    render(<Footer />);
    const copyrightText = screen.getByText(
      /Â© 2023 CycleSync. All rights reserved./i
    );
    expect(copyrightText).toBeInTheDocument();
  });

  it.skip("Should render 1 p tag", () => {
    render(<Footer />);
    const pTag = screen.getByRole('p');
    expect(pTag).toBe(1);
  });
});
