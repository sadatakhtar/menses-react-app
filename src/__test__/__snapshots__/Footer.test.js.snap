// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Footer snapshot created without errors 1`] = `
<div>
  <div>
    <footer>
      <p>
        Â© 2023 CycleSync. All rights reserved.
      </p>
    </footer>
  </div>
</div>
`;
